libs/common/src/pubsub/pubsub.service.ts

```diff
@@ -55,6 +56,10 @@ export class PubSubService {
             this.logger.error(`Failed to send announcement to webhook: ${webhookUrl}`);
             this.logger.error(error);
             retries += 1;
+            // eslint-disable-next-line no-await-in-loop
+            await new Promise((r) => {
```

Comment:
seems a bit out of scope. I think it might be a good idea to separate this and write a test for it.

Reply:
Part of the purpose of this exercise is to make all of our gateway services configurable in the same standard ways. Our other gateway services that interact with webhooks all honor this environment variable; rather than delete it as unused, it seemed logical to add in support for the delay. Plus hammering a failed webhook with no delay between retries is not good practice.

Additional considerations such as exponential backoff, etc, would be more complex and can be deferred to a separate issue, but since this is a one-liner, it seems appropriate.

Reply:
Yeah. I do agree that it's needed. I think one of the concerns that has been brought during our retro was adding test. I think this one of those cases that we can be proactive to ensure the quality of our code. I do recommend adding a ticket for this so that it does not go unnoticed. Please try to keep things scoped to your ticket.